publisher-sdk/src/test/java/com/criteo/publisher/BidPrefetchRateLimiterTest.kt

```diff
@@ -0,0 +1,100 @@
+/*
+ *    Copyright 2020 Criteo
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+package com.criteo.publisher
+
+import android.content.SharedPreferences
+import com.criteo.publisher.BidPrefetchRateLimiter.Companion.STORAGE_KEY
+import com.criteo.publisher.BidPrefetchRateLimiter.Companion.UNSET_PREFETCH_TIME
+import com.criteo.publisher.model.Config
+import com.criteo.publisher.util.SafeSharedPreferences
+import com.nhaarman.mockitokotlin2.verify
+import com.nhaarman.mockitokotlin2.whenever
+import org.junit.Before
+import org.junit.Test
+import org.mockito.InjectMocks
+import org.mockito.Mock
+import org.mockito.MockitoAnnotations
+import kotlin.test.assertFalse
+import kotlin.test.assertTrue
+
+class BidPrefetchRateLimiterTest {
```

Comment:
Do you think it might be worth to add a test checking for the persistence ? Or are we confident enough on the shared pref ?